packages/test/test-end-to-end-tests/src/test/incrementalSummaryContext.spec.ts

```diff
@@ -257,9 +256,11 @@ class TestIncrementalSummaryTreeDDS extends SharedObject {
 	static getFactory(): IChannelFactory {
 		return new TestTreeDDSFactory();
 	}
+
+	public readonly rootNodeName = "rootNode";
```

Comment:
Moved this inside the DDS so that this is not used outside of it. I got confused while adding validations since the two DDSes here are named so similar.

Reply:
looks good